src/CorsMiddleware.php

```diff
@@ -31,33 +28,30 @@ public function __construct(ConfigurationLoader $configurationLoader) {
      * @param Request $request
      * @param RequestHandler $requestHandler
      *
-     * @return Promise<Response>
+     * @return Response
```

Comment:
_nitpick_ We can actually just drop all the docblocks specifying types here. They're no longer necessary with the return type specifying `Response`. Normally I wouldn't hold up merging this in because of this but since making the change to composer.json anyway and this is a small one if you wouldn't mind, please. :)

Reply:
done!